docs/reference/cassandra-schema.cql

```diff
@@ -311,17 +347,14 @@ CREATE TABLE galley_test.user (
     AND read_repair_chance = 0.0
     AND speculative_retry = '99PERCENTILE';
 
-CREATE TABLE galley_test.member_remote_user (
-    conv uuid,
-    user_remote_domain text,
-    user_remote_id uuid,
-    conversation_role text,
-    PRIMARY KEY (conv, user_remote_domain, user_remote_id)
-) WITH CLUSTERING ORDER BY (user_remote_domain ASC, user_remote_id ASC)
-    AND bloom_filter_fp_chance = 0.1
+CREATE TABLE galley_test.group_id_conv_id (
+    group_id blob PRIMARY KEY,
+    conv_id uuid,
+    domain text
+) WITH bloom_filter_fp_chance = 0.01
```

Comment:
I tried to reproduce this locally by compiling your branch, doing a db-migrate, then a make git-add-cassandra-schema; I get the same ordering you get (i.e. the one with the big diff here). There's also https://issues.apache.org/jira/browse/CASSANDRA-14534 which indicates there is no guarantee for ordering of these objects.
I'd just go with what you have, i.e. the bigger diff. The actual changes can be seen in the schema migrations, this file can be seen as a "state-of-now" snapshot, rather than a useful too to see changes over time, for that we have the schema migrations.